common/display/virtualdisplay.cpp

```diff
@@ -308,6 +309,13 @@ bool VirtualDisplay::Present(std::vector<HwcLayer *> &source_layers,
           }
           mHyperDmaExportedBuffers.erase(search);
         }
+
+        FrameBufferManager *fb_manager_ =
+            GpuDevice::getInstance().GetFrameBufferManager();
+
+        fb_manager_->RemoveFB(handle.handle_->meta_data_.num_planes_,
```

Comment:
Suggesting checking fb_manager_ is not NULL before referencing it.

Reply:
Yes, make sense. Will update the PR accordingly.